pkg/cmd/roachtest/test_registry.go

```diff
@@ -81,8 +81,7 @@ var roachtestOwners = map[Owner]OwnerMetadata{
 	OwnerPartitioning: {SlackRoom: `kv`, Mention: []string{`@cockroachdb/kv`},
 		TriageColumnID: 3550674,
 	},
-	// TODO(jordanlewis): that team doesn't exist any more, fix things up.
-	OwnerSQLExec: {SlackRoom: `sql-execution-team`, Mention: []string{`@cockroachdb/sql-queries`},
+	OwnerSQLQueries: {SlackRoom: `sql-queries`, Mention: []string{`@cockroachdb/sql-queries`},
 		TriageColumnID: 6837155,
```

Comment:
this should be `13549252` 

based on this link: https://github.com/cockroachdb/cockroach/projects/61#column-13549252